signer/src/transaction_coordinator.rs

```diff
@@ -0,0 +1,61 @@
+//! # Transaction coordinator
+//!
+//! This module contains the transaction coordinator, which is the component of the sBTC signer
+//! responsible for consctructing transactions and coordinating signing rounds.
+//!
+//! For more details, see the [`EventLoop`] documentation.
+
+#[cfg_attr(doc, aquamarine::aquamarine)]
+/// # Transaction coordinator event loop
+///
+/// This struct contains the implementation of the transaction coordinator logic.
+/// Like the transaction signer, the coordinator event loop also subscribes to storage
+/// update notifications from the block observer and listends to signer messages over
+/// the signer network.
+///
+/// The transaction coordinator will upon receving a storage update notification from
+/// the block observer look up the canonical chain tip from the database. This tip
+/// is used to decide wheter or not this particular coordinator is selected to be active
+/// or if it should be passive in favor of another coordinator in the signer network.
```

Comment:
```suggestion
/// The transaction coordinator will look up the canonical chain tip from
/// the database upon receiving a storage update notification from the
/// block observer. This tip is used to decide whether this particular
/// signer is selected as the signers' coordinator or if it should be
/// passive in favor of another signer as the coordinator in the signer
/// network.
```
This looks like more of an edit than it actually is since I changed line length for readability on Github.